### ISSUE TITLE
Offline Installation of IRkernel

### ISSUE THREAD

**Issue by GokulRamE** [IRkernel/IRkernel#488]
I am having issues with installation of IRKernel through the install_github function hence had to try offline installation wherein I downloaded the "IRkernel-master.zip" extracted it and used both the following codes to install

```r
install_local("C:\\Users\\gokul.e\\Documents\\IRkernel-master")
install("C:\\Users\\gokul.e\\Documents\\IRkernel-master")
```

Both of these give the same error mentioned below

```r
Installing IRkernel
"C:/PROGRA~1/R/R-34~1.1/bin/x64/R" --no-site-file --no-environ --no-save  \
  --no-restore --quiet CMD INSTALL "C:/Users/gokul.e/Documents/IRkernel-master"  \
  --library="C:/Users/gokul.e/Documents/R/win-library/3.4" --install-tests 

* installing *source* package 'IRkernel' ...
** help
No man pages found in package  'IRkernel' 
*** installing help indices
** building package indices
** testing if installed package can be loaded
*** arch - i386
Error: package or namespace load failed for 'IRkernel' in namespaceExport(ns, exports):
 undefined exports: comm_manager, installspec, jupyter_option_defaults, log_debug, log_error, log_info, main
Error: loading failed
Execution halted
*** arch - x64
Error: package or namespace load failed for 'IRkernel' in namespaceExport(ns, exports):
 undefined exports: comm_manager, installspec, jupyter_option_defaults, log_debug, log_error, log_info, main
Error: loading failed
Execution halted
ERROR: loading failed for 'i386', 'x64'
* removing 'C:/Users/gokul.e/Documents/R/win-library/3.4/IRkernel'
Error: Command failed (1)
````

**Comment by flying-sheep** [comment]
hmm “No man pages found in package  'IRkernel'”: we definitely have man pages.

somehow you don’t have the right files in that directory.

**Comment by GokulRamE** [comment]
You are correct,

Actually, when I extracted the .zip which I downloaded from github the folder was extracted in two levels. Hence, the correct 'IRkernel-master' (with a folder called 'man') folder was present inside the 'IRkernel-master' which I had given for source in my earlier code. Installation worked when I edited the code as below

```
install("C:\\Users\\gokul.e\\Documents\\IRkernel-master\\IRkernel-master")
```

Didn't know that this would happen. Thank You very much @flying-sheep
